### PR TITLE
Works and about sections

### PR DESCRIPTION
**Pull request by Ninjaphil24**
Good evening.  This has been a very complicated venture for the first week.  
I have simply continued from where I left off the previous project, so it's in effect one file that has a continuous flow, like in the figma template.  

This was the first part of the project:
image

Subsequently, as part of the same .html and .css files, I have added the following pages:
image
image
image
image

There is a linter problem that I cannot understand.  It says the following:

index.css
 168:1  ✖  Expected selector ".page3 .buttontypeA a" to come before       no-descending-specificity 
 197:1  ✖  Expected selector ".page2 .buttontypeB a" to come before       no-descending-specificity 
           selector ".page2 .buttontypeA a:hover"
 197:1  ✖  Expected selector ".page2 .buttontypeB a" to come before       no-descending-specificity 
           selector ".page3 .buttontypeA a:hover"
 246:1  ✖  Expected selector ".page3 .text1" to come before selector      no-descending-specificity 
           ".page2 .site1 .text1"

This makes little sense though.  the sequence in my view is as it should be.  I don't know what to do about this linter problem.  It feels like a glitch to me.  Let me know what you think.

Best,
Philip

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
             <li><a href="https://www.linkedin.com/in/philip-modinos-14195021"><i class="fa fa-linkedin-square fa-3x" aria-hidden="true"></i></a></li>
         </ul>
     </aside>   
+    
+</div> <!--page 1 end -->
 
+<div class="page2">
```
Comment (Lordkaito) [inline index.html]: - [ ] Please make sure to use a `<section>` tag to divide your document. This is a good practice as well as it helps to keep your code more semantic, so any other developer, or even yourself, can understand it in the future if needed.

File: index.html [hunk 2]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</div><!--page2 end-->
+<br> <br> <br>
+<div class="page3">
```
Comment (Lordkaito) [inline index.html]: - [ ] As I mentioned earlier, it is good to use `<section>` tags to divide your document instead of a pure `<div>`, so please make sure to apply it here too.

File: index.css [hunk 3]
```diff
@@ -223,9 +343,117 @@ header li a:hover {
   .asidemobile ul li i {
     color: #cfcfcf;
   }
+
+  .page2 .container {
+    display: grid;
+    grid-template-rows: auto auto auto auto auto;
```
Comment (Lordkaito) [inline index.css]: - [ ] [OPTIONAL] Instead of repeating the same `auto` word 5 times, you can use the `repeat()` CSS function in order to achieve the same, so it will help your code to be cleaner. If you want more information, you can learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat()).

File: index.html [hunk 4]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
         </ul>
     </aside>   
+    
+</div> <!--page 1 end -->
 
+<div class="page2">
+    <div class="container">
+        <div class="items title1">
+            My Recent Works        
+        <br> <br>
```
Comment (Lordkaito) [inline index.html]: - [ ] It is important to keep your code semantic, because this is helpful for you to understand your code if you have to read it again later, so please make sure to use `<h1>` or any of the other `<h2>`,`<h3>`,`<h4>`,`<h5>`,`<h6>` tags to write titles in your document. So kindly add any of them here, you can choose the one that fits better for you!

File: index.html [hunk 5]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+        <hr>
+        </div>
+        <div class="items img1">
+            <img src="images/testimg.png" alt="">
+        </div>
+        <div class="items description1">
+            <div class="title2">Multi-Post Stories</div>
+            <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
```
Comment (Lordkaito) [inline index.html]: - [ ] When adding text to your document that it's not a title, please make sure to use a `<p>` tag instead of a `<div>`. Please apply this to all the rest of the sections where you can spot this mistake.

File: index.html [hunk 6]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
+            
+        </ul>
+        <div class="items buttontypeA">
+        <a href="#">See Project</a>
+        </div>
```
Comment (Lordkaito) [inline index.html]: - [ ] Please make sure to use Flexbox to align each of the "cards" from your Work Section. Using Flexbox combined with CSS Grid is one of the requirements that you have to accomplish in order to be able to move to the next project.

File: index.html [hunk 7]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+            <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+        </div>
+        <ul class="items lang">
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
+            
+        </ul>
```
Comment (Lordkaito) [inline index.html:78]: - [ ] Please try to keep your code without blank lines that have no purpose as this is part of [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and those are a requirement for your project.

File: index.html [hunk 8]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+        <div class="items description1">
+            <div class="title2">Multi-Post Stories</div>
+            <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+        </div>
+        <ul class="items lang">
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
```
Comment (Lordkaito) [inline index.html:76]: - [ ] For the list of languages in your "cards", please try to keep the same amount of languages as in the original design. Personally, I like the idea of adding all the languages you know, but they are breaking the original design, and one of your requirements is to build the page as similar as possible to the original one.

File: index.css [hunk 9]
```diff
@@ -133,6 +138,121 @@ header li a:hover {
...
+}
+
+.page2 .site2 {
+  background-image: url(images/spoiledeggs1.png);
+}
+
+.page2 .buttontypeB {
+  margin-top: 10px;
+  text-align: center;
+}
```
Comment (Lordkaito) [inline index.css:207]: - [ ] Please take care of margins on your "See project" button. It is separated from the rest of the card, but the original design does not have the button separated. 

Yours: 
Captura de pantalla 2022-01-06 190218

Figma template:
Captura de pantalla 2022-01-06 190242

File: index.html [hunk 10]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</div><!--page2 end-->
+<br> <br> <br>
+<div class="page3">
+    <div class="container">
+        <br> <br> 
+        <div class="items title1">About me</div>
```
Comment (Lordkaito) [inline index.html]: - [ ] As I mentioned earlier, please make sure to apply an `<hx>` tag where a title is required. This is to improve your code to be more semantic.

File: index.html [hunk 11]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+        <div class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (Lordkaito) [inline index.html]: - [ ] Kindly put this paragraph into a `<p>` tag. This will help you to be more semantic.

File: index.html [hunk 12]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+        <div class="items buttontypeA">
+            <a href="#">Get My Resume</a>
+        </div>
+        <div class="items graphic">
+            <img src="images/graphic.png" alt="">
+        </div>
+        <div class="items hr">
+            <hr>
+        </div>
+        <div class="items languages">
```
Comment (Lordkaito) [inline index.html]: - [ ] Kindly change this `<div>` container into a `<ul>` for all "Languages", "Frameworks", and "Skills" and use the proper `<li>` tag for the correct elements. As a result, you will have a nested `<ul>` (an `<ul>` inside another `<ul>`).

File: index.html [hunk 13]
```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
...
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</div><!--page2 end-->
+<br> <br> <br>
```
Comment (Lordkaito) [inline index.html:120]: - [ ] As I mentioned earlier, Flexbox is a requirement for this project, so please make sure to use it for your section. You can use CSS Grid for the `<section>` container, but make sure to use Flexbox for the elements inside the `<section>`.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Ninjaphil24 , 

Good job so far!
There are some issues that you need to fix before moving to the next project. Keep it up!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] You can add `node_modules/` folder to the `.gitignore` file. This is not a major requirement, but it is a good practice, because sometimes you may include the `node_modules/` folder into your repository by accident when setting up your linters.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ninjaphil24** [comment]
section tags - Done.
Add node_modules/ to .gitignore - Done
index.css line 349 Use repeat(5, auto), done 
Conflicting reviews:  h1, h2 tags etc:  I used that in the first page of the project and was told to remove it by the first reviewer.  I generally practice the use of hx tags, and opted not to do so here because of the first review I got.  

On the issue of hx tags, I request that an exception be made for index.html lines 84 and 102, the reason being that replacing or adding those tags at this point completely ruins the styling.  In a future iteration of the project I will correct the matter.  

Other than that all other hx and p tags: Done.

"See project" button gap (index.css line 203 done)

Nested ul and li - Done

Concerning the matter of using flex and grid. 

I have used flex in 31 instances one way or another within this document.
Furthermore, the entire document is a grid as can be seen from the following print screen from google dev tools

image


image

But also flex:

image

More grid:

image


In short, the entire document from page 2 is a combination of grid and flex.  I don't know what else to add that will make it more grid and flex and as such, believe that there is no correction necessary in this regard.  

Thanks for all the input, it was a great help!!

File: index.html [hunk 14]
```diff
@@ -9,9 +9,9 @@
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
 </head>
 <body>
-<div class="page1">
+<section class="page1">
     <header>
-    <a href="index.html">Philip Modinos</a>
+    <h1>Philip Modinos</h1>
```
Comment (HENRYKC24) [inline index.html]: - [ ] Good job on trying to make some changes to your project. However, you seem to have broken something here. I think you forgot to wrap the `h1` element with an anchor (`a`) tag. This will help in making your name a link.

**Comment by Lordkaito** [comment]
Hi @Ninjaphil24,

You did a really good job not only implementing changes but also explaining them here as well. 

- Concerning the CSS Grid and Flex requirement, I think this is good enough to go, but unfortunately, I cannot change the review that's already done. Please go ahead with a new review request, the next reviewer will see these comments and I'm sure he will take care of it. 

- Concerning the hx tags on lines 84 and 102, and as I mentioned before, I cannot modify a review that's already done, so please ask for a new review, and before submitting, in the text box that you have, bring this information to the next code reviewer, so he can take a look at it and approve it if he thinks it's good to go.

- And lastly, concerning the previous reviews that you received, I'm not sure why the previous reviewer asked you to remove the hx tags that you already used, as those tags are part of the requirements for this project. I apologize for the inconvenience. I'll make sure to always make my reviews explaining in detail why I'm requesting the changes, and always sticking to the requirements for the project.

Thanks for your comments, good luck with the upcoming reviews!

Cheers and happy coding! :shipit:

File: index.html [hunk 15]
```diff
@@ -51,7 +51,115 @@ <h3>I'm a Software Developer</h3>
...
+        <br> <br>
+        <hr>
+        </div>
+        <div class="items img1">
+            <img src="images/testimg.png" alt="">
+        </div>
+        <div class="items description1">
+            <h3 class="title2">Multi-Post Stories</h3>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
```
Comment (HENRYKC24) [inline index.html:70]: [OPTIONAL] I think you need to stick to the design template as much as possible. There are some discrepancies between your Design and that of Figma and it will enhance the quality of your portfolio to get them fixed.
## Your Design:point_down:
image

## Figma Design:point_down:
image

File: index.html [hunk 16]
```diff
@@ -51,7 +51,115 @@ <h3>I'm a Software Developer</h3>
...
+        <br> <br> 
+        <h3 class="items title1">About me</h3>
+        <p class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="items buttontypeA">
+            <a href="#">Get My Resume</a>
+        </div>
+        <div class="items graphic">
+            <img src="images/graphic.png" alt="">
+        </div>
```
Comment (HENRYKC24) [inline index.html]: [OPTIONAL] Let's try to fix these design issues. You seemingly forgot to use the same padding and margins as specifiedd in the Figma file.
## Your Design
image

## Figma Design
image
Please, try to space out the button with the same margin as we have it in the Figma file.

File: index.html [hunk 17]
```diff
@@ -9,9 +9,9 @@
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
 </head>
 <body>
-<div class="page1">
+<section class="page1">
```
Comment (HENRYKC24) [inline index.html:12]: [OPTIONAL] Let's try to stick to the font, font size, margins, and paddings as it is in the Figma Template.
The left padding in your design is too small that the text is very close to the left edge of the page. **Please, fix this I advise**.
image
The font size of the header text is also not as big as that of Figma's.
## Figma's
image

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED:recycle:
Close
## Hi @Ninjaphil24 :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing the requested changes, but you broke something else.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ninjaphil24** [comment]
Dear 3rd reviewer.  This is my final attempt on this project, so I would greatly appreciate you passing me.  Here are the things I did:
a tags - done 
space issues - done
font and margins - done

Note about the fonts.  Here is a screenshot example of the font from figma for the title:

image

You will notice that I'm using the same font, but adding "serif" afterwards.
image


I'm doing this on the advice of another reviewer, who in this way assisted me in resolving a linter issue of external font families.  I have used all the fonts from figma, but must use the serif next to it so that it passes stylelint.  

Furthermore, please note that the first page of the file (the one containing the title "Hey there, I'm Phil") is not a part of this project.  That page had already been passed.  I don't mind doing changes on it, don't get me wrong, I just feel that this represents conflict in the reviews.  In any case, the changes suggested were good, so I did them.  

So, hoping you'll pass the project, as I've done everything humanly possible as requested by the other reviewers.  

Best,
Philip

File: index.html [hunk 18]
```diff
@@ -51,7 +51,115 @@ <h3>I'm a Software Developer</h3>
...
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</section><!--page2 end-->
+<br> <br> <br>
+<section class="page3">
+    <div class="container">
+        <br> <br> 
+        <h3 class="items title1">About me</h3>
+        <p class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
```
Comment (VICTOR-CHUKWU) [inline index.html:125]: - [OPTIONAL] kindly align your `h3` tag so that it does not overlap with your `p` tag
  Your design 👇 
              
image
Figma design 👇 
        
image

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Ninjaphil24  👏 , 
## Status: Approved 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
